docs/next/components/mdx/MDXComponents.tsx

```diff
@@ -337,10 +338,34 @@ const Experimental = () => {
   );
 };
 
-const Pre = ({children, ...props}) => {
+// next-mdx converts code blocks to <code> elements wrapped in <pre> elements.
+// We need to access the props of the code element to see if we should render an asset
+// graph DAG; so we turn the <pre> tag into a styling no-op and create a new <pre> tag
+// around the <code> tag once we have access to the <code> tag's props.
+const Pre: React.FC<React.HTMLProps<HTMLPreElement>> = ({children, ...props}) => {
+  const updatedProps = {...props, className: 'noop'};
+  return (
+    <pre {...updatedProps}>
+      {React.Children.map(children, (child: ReactElement<any>) =>
```

Comment:
First part of fix; attaches new prop to children of `Pre` tag to let them know they're `Pre` children.